real_intent/internal_logging.py

```
@@ -37,6 +37,26 @@ def disable_logging():
+# ---- Instrumentations ----
+
+def instrument_openai(openai_client=None):
+    """Instrument the OpenAI client.
+
+    Raises:
+        ImportError: If the OpenAI package is not installed.
+    """
+    try:
+        from openai import OpenAI
+    except ImportError:
+        raise ImportError("Please install this package with the 'ai' extra.")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)